src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -67,8 +77,19 @@ export class MarkdownTextTransformer {
 				endLineNumber: selection.startLineNumber
 			};
 
-			let beginInsertedText = getStartTextToInsert(type);
-			let endInsertedText = getEndTextToInsert(type);
+			let beginInsertedText: string;
+			let endInsertedText: string;
+			if (type === MarkdownButtonType.IMAGE) {
```

Comment:
Shouldn't this logic go in the getStartTextToInsert function instead? Seems like that's where we'd want to keep all this kind of logic so everything is consolidated.

Reply:
@Charles-Gagnon --> This is a question for @corivera as he had implemented this.